content_dev/docs/sdk/java/introduction/atprotocol.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: codelab
+
+title: atProtocol # Step Name
+description: | # SEO Description for this step
+  What is the atProtocol?
+
+draft: false # TODO CHANGE THIS TO FALSE WHEN YOU ARE READY TO PUBLISH THE PAGE
+order: 2 # Ordering of the steps
+---
+
+## Overview
+The atProtocol lets you communicate with the root server and secondary server allowing you to do some pretty amazing things like sending encrypted data to other people. 
```

Comment:
Need to workshop this message a bit.

Reply:
5cb13e049ea795ebccea81420dcca070658f4edb